fixtures/uking/mini-spider-light.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Spider Light",
+  "shortName": "UK-MiniSpider",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Anonymus", "CGBassPlayer"],
```

No need to credit someone who doesn't want to be credited.

```suggestion
    "authors": ["CGBassPlayer"],
```